code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -628,7 +628,6 @@ Assign your candidates in choose_candidates() instead.
 	role_category = null
 	restricted_from_jobs = list()
 	enemy_jobs = list()
-	required_pop = list(30,30,30,30,30,30,30,30,30,30)
```

Comment:
I'd add a comment in the entry here to explain that there's a custom ready() and that the normal variables do not apply.